index.js

```diff
@@ -0,0 +1,183 @@
+  }
+});
+
+const speakersArray = [
+  {
+    name: 'Sarah smith',
+    title: 'Professor of IT UoN university',
+    profilepicture: 'https://randomuser.me/api/portraits/women/62.jpg',
+    bio: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum porro voluptate ullam corporis assumenda iste laborum perspiciatis. Esse, minima dicta.',
+  }, {
+    name: 'John Doe',
+    title: 'Director of Computer Science MMUT University',
+    profilepicture: 'https://randomuser.me/api/portraits/men/5.jpg',
+    bio: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum porro voluptate ullam corporis assumenda iste laborum perspiciatis. Esse, minima dicta.',
+  }, {
+    name: 'Tom Harry',
+    title: 'It Expert',
+    profilepicture: 'https://randomuser.me/api/portraits/men/10.jpg',
+    bio: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum porro voluptate ullam corporis assumenda iste laborum perspiciatis. Esse, minima dicta.',
+  }, {
+    name: 'Jane Doe',
+    title: 'Lecturer MKU University',
+    profilepicture: 'https://randomuser.me/api/portraits/women/12.jpg',
+    bio: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum porro voluptate ullam corporis assumenda iste laborum perspiciatis. Esse, minima dicta.',
+  }, {
+    name: 'Joe Blogs',
+    title: 'Networking and System Security Expert',
+    profilepicture: 'https://randomuser.me/api/portraits/men/12.jpg',
+    bio: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum porro voluptate ullam corporis assumenda iste laborum perspiciatis. Esse, minima dicta.',
+  }, {
+    name: 'Jane Smith',
+    title: 'Data Analyst',
+    profilepicture: 'https://randomuser.me/api/portraits/women/2.jpg',
+    bio: 'Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum porro voluptate ullam corporis assumenda iste laborum perspiciatis. Esse, minima dicta.',
+  },
+];
+
+const cardContainer = document.createElement('div');
+cardContainer.className = 'card-container';
+
+const speakers = Object.keys(speakersArray);
+
+speakers.forEach((speaker) => {
+  const card = document.createElement('div');
+  card.className += 'd-flex card';
+  const cardImg = document.createElement('div');
+  const cardInfo = document.createElement('div');
+  cardInfo.className = 'd-flex-column cardinfo';
+  const name = document.createElement('h2');
+  name.innerHTML = speakersArray[speaker].name;
+  const title = document.createElement('h3');
+  title.className = 'heading';
+  title.innerHTML = speakersArray[speaker].title;
+  const profilepicture = document.createElement('img');
+  profilepicture.setAttribute('src', speakersArray[speaker].profilepicture);
+  const bio = document.createElement('p');
+  bio.className = 'bio';
+  bio.innerHTML = speakersArray[speaker].bio;
+
+  cardImg.appendChild(profilepicture);
+  cardInfo.append(name, title, bio);
+  card.append(cardImg, cardInfo);
+  cardContainer.appendChild(card);
+  sec.appendChild(cardContainer);
+});
```

Comment:
Good job making the section dynamic 👍
- [ ] Please consider using the window.onload function for this, so that if their is slow internet or in any similar case the script doesn't execute before actual html page is rendered. It is kindly requested to run speakers.forEach function inside window.onload to avoid such circumstances. I know you can do that 💪